components/NewsletterForm/NewsletterForm.tsx

```diff
@@ -90,12 +91,18 @@ export default function NewsletterForm() {
             onChange={handleChange}
           />
         </label>
-        <button
-          className={clsx(styles.submit, 'mt-6 lg:col-span-2')}
-          type="submit"
-        >
-          Submit
-        </button>
+        {state === FormState.Loading ? (
+          <div className={clsx(styles.loader, 'mt-6 lg:col-span-2')}>
+            <ClipLoader size={35} color={'#00704a'} />
+          </div>
+        ) : (
+          <button
+            className={clsx(styles.submit, 'mt-6 lg:col-span-2')}
+            type="submit"
+          >
+            Submit
+          </button>
```

Comment:
Could this loader potentially go in the button instead? So something like:
```suggestion
          <button
            className={clsx(styles.submit, 'mt-6 lg:col-span-2')}
            type="submit"
          >
            {state === FormState.Loading ? <ClipLoader size={35} color={'#00704a'} /> : "Submit"}
          </button>
```

I suppose this is up to the designers, but would be good to not have to duplicate the button style.

Reply:
Ooo okay, sure!